### DISCUSSION TITLE
How to upload multiple images to component media field specifying photo order

### DISCUSSION THREAD

**Discussion by mscalessio** [strapi/strapi#7735] [Legacy Topics]
Hi there,
I am following this guide https://strapi.io/documentation/v3.x/plugins/upload.html#upload-file-during-entry-creation.
Everything is fine and I am able to upload multiple file in a media field inside a component.

When i create a post data request with the following order of the images:
`files.images: image-a.jpg`
`files.images: image-b.jpg`

In the dashboard i found this order in the carousel:
`image-a.jpg, position: 2`
`image-b.jpg, position: 1`

To be secure I would like to assign a specified position to a specific image.
Is that possible?

**Comment by derrickmehaffy** [comment]
Currently it is not possible to define the order of media fields, as media entries are polymorphic in nature they could be in an entirely different order when used in another model. Only the components themselves can maintain order at the moment, we have an entry on our FAQ to explain this: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#relations-aren-t-maintaining-their-sort-order
